task/run.go

```diff
@@ -24,6 +24,137 @@ func Run(registry *Registry, arguments []string) error {
 		return err
 	}
 
+	if _, ok := opts.args.get("", "json"); ok {
+		return runWithJSONOutput(registry, opts)
+	}
+
+	return runWithHumanOutput(registry, opts)
+}
+
+func runWithJSONOutput(registry *Registry, opts *runOptions) error {
```

Comment:
My only qualm here is there is duplication with `runWithHumanOutput`, but there wasn't a super-clean way at first glance to generalize the human-readable output